lms/djangoapps/courseware/views.py

```diff
@@ -726,6 +726,23 @@ def registered_for_course(course, user):
         return False
 
 
+def get_cosmetic_display_price(course, registration_price):
+    """
+    Return Course Price as a string preceded by correct currency, or 'Free'
+    """
+    currency_symbol = settings.PAID_COURSE_REGISTRATION_CURRENCY[1]
+
+    price = course.cosmetic_display_price
+    if registration_price > 0:
+        price = registration_price
+
+    if price:
+        return "{}{}".format(currency_symbol, price)
```

Comment:
@sarina, I'm not sure if translating is the right way to change the display of currency...shouldn't $50 always be displayed as $50, whether in French, Spanish, Japanese, or English? 

I don't mind making that change though is that's what you guys want...